.devcontainer/Dockerfile

```diff
@@ -0,0 +1,86 @@
+ARG DEBIAN=debian
```

Comment:
I wonder if it might make sense to split this Docker / devcontainer stuff out into a separate PR, so that it can be reviewed and merged independently of this one?

Reply:
Sure - that would also give a chance to consider prebuilding the docker images, providing them from ghcr.io and adding some more info in the docs. It's only valuable to those who want to develop or extend gpiozero ...
I was using it to check compatibility with each specific install but none of the below comments look like there is much risk. And I can remove it as a last step to finishing this PR anyway...

Reply:
@bennuttall , @waveform80: What is your view on this?
IMO the ability to code, debug and test in the specific versions is part of supporting them - and I can't think of a better way of enabling that than with devcontainers for oldoldstable through to stable. Having the github workflow run on all supported python versions is good practice to help ensure that no future changes break backward compatibility, but doesn't necessarily help enough when fixing any issues which it throws up...

If you leave this in would you like more docs and prebuilt docker images as well?